pkgs/http/lib/src/byte_stream.dart

```diff
@@ -7,7 +7,7 @@ import 'dart:convert';
 import 'dart:typed_data';
 
 /// A stream of chunks of bytes representing a single piece of data.
-class ByteStream extends StreamView<List<int>> {
+final class ByteStream extends StreamView<List<int>> {
```

Comment:
(This class should not need to exist. It should just be extension methods on `Stream<List<int>>`.)